### ISSUE TITLE
"Attempt to use history.replaceState() more than 100 times per 30 seconds" bugs out whole interface across multiple machines

### ISSUE THREAD

**Issue by peace-blaster** [cockpit-project/cockpit#14721]
Using Safari on MacOS as my browser, in case that matters.

When attempting to connect a RHEL VM (on AWS) to a local machine, the console asked me to log in an additional time, and I got the following error:

"Attempt to use history.replaceState() more than 100 times per 30 seconds"

This then caused the same error with the tabs to all connected machines, and persisted through refreshing tabs, and even rebooting individual machines. It was only after I rebooted every machine added in Cockpit that the error went away, and I could resume normal use.

**Comment by rasmuslos** [comment]
+1

**Comment by marusak** [comment]
Duplicate of #14936
